### ISSUE TITLE
Terminology for configuring a block seems odd

### ISSUE THREAD

**Issue by Graham-72** [backdrop/backdrop-issues#1016]
When adding a block into a layout I was initially puzzled by 'Title type'
![screenshot1](https://cloud.githubusercontent.com/assets/10559274/8268406/d3670d1a-177c-11e5-9140-d6df4dfcfb7d.png)
Why is it called 'type'? Why not 'Title display' because that is what we are controlling here.

Also, then there is the question about a 'dynamic' class
![screenshot2](https://cloud.githubusercontent.com/assets/10559274/8268425/136544d6-177d-11e5-8d6a-be4285fb22e7.png)
Would it not be easier to understand if the option was 'Custom'?

**Comment by klonos** [comment]
Agreed on both points here. Perhaps "default"/"custom"/"hidden" for title and "default"/"custom" for style.

**Comment by klonos** [comment]
...and we should standardize on either using "Default"/"Custom"/"Nome" (currently used in the blocks UI) or "Default title"/"Custom title"/"No title" (layouts UI):

![layout-title_display_options](https://cloud.githubusercontent.com/assets/2423362/8270573/4be5e22e-1828-11e5-9d8d-dc84b9ba9757.png)

Either way is fine (I personally like the second), lets just be consistent.

**Comment by klonos** [comment]
There was an issue in d.org about replacing the "<none>" thing with a "Hide" checkbox or something like that, but I can't seem to be able to find it...

**Comment by klonos** [comment]
Oh, turns out I've already filed an issue about this... #692

**Comment by Graham-72** [comment]
It was the term 'Title type' that confused me at first sight. I started wondering what different types of title exist, then I realised it refers to the display of the title. I don't mind whether the option to hide it is 'none' or 'hide'.

**Comment by klonos** [comment]
Yep, I fully agree. It's just the same things that bothered me before filing #692, that if implemented will render any refactoring here redundant. For example, the whole drop-down might go away if...

a) the "no title"/"none"/"hidden" becomes a "Display title" checkbox.
b) the "Default" is not an option (as in an editable field at all) - it is simply displayed in plain/rendered text.
c) the "Custom" becomes an "edit" link next to the default title text that makes the default title an editable field.

**Comment by Graham-72** [comment]
Sounds good :+1:

**Comment by quicksketch** [comment]
Sounds good to me too, but I'm open to fixing the terminology first rather than trying the more extensive solution of breaking this into multiple fields.
